cmd/istio-cni/redirect.go

```diff
@@ -64,6 +63,15 @@ var (
 	}
 )
 
+// Tables defines interface which must be implemented by tables mechanism driver (ex. ip tables )
+type Tables interface {
```

Comment:
I'd prefer a different interface name.  My preference would be something like "Redirector" or "RedirectDriver"

Reply:
@tiswanso the reason I did not go for Redirector name is because it is too specific, if in future you want to add other features of Tables like connection tracking or some counters, then adding then to Redirector will be confusing, whereas adding them to Tables interface by means of Program method seems logical. If my arguments are not strong enough I can change interface name. Please let me know.

Reply:
Changed to `InterceptRulesMgr`